src/main/java/racingcar/StringAddCalculator.java

```diff
@@ -0,0 +1,78 @@
+package racingcar;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static int splitAndSum(String inputString) {
+
+        String delimiter = getDelimiter(inputString);
+
+        String[] result = splitString(delimiter, inputString);
+
+        return sumStrings(result);
+    }
+
+    private static Matcher getMatcher(String inputString) {
+
+        Matcher matcher = null;
+
+        if (inputString != null) matcher = Pattern.compile("\\/\\/(.+)\\\\n(.+)").matcher(inputString);
+
+        return matcher;
+    }
+
+    private static String getDelimiter(String inputString) {
+
+        String delimiter = "[,:]";
+
+        Matcher matcher = getMatcher(inputString);
+        if (matcher != null && matcher.find()) delimiter = matcher.group(1);
+
+        return delimiter;
+    }
```

Comment:
null or empty 체크를 앞에서 한 번 했기 때문에 이후 메서드에서 관련 로직을 제거할 수 있어요.

```suggestions
    private static String getDelimiter(String inputString) {
        String delimiter = "[,:]";
        Matcher matcher = getMatcher(inputString);
        if (matcher.find()){
            delimiter = matcher.group(1);
        }
        return delimiter;
    }
```